hack/run_test

```diff
@@ -372,15 +376,19 @@ function run_master_restart_test() {
   local PASS=master
 
   echo "Testing failed master restart"
-  local cluster_args="-e POSTGRESQL_ADMIN_PASSWORD=pass -e POSTGRESQL_MASTER_USER=$PGUSER -e POSTGRESQL_MASTER_PASSWORD=$PASS"
   local cid_suffix="mrestart"
+  local network=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
> running the same test case for different versions of postgresql in parallel 

Good point.  How that is possibe that the tests are run in parallel now in RHEL CI?  The makefile was not ready for that yet, when I was looking at it last time.